atom/common/options_switches.cc

```diff
@@ -110,6 +110,9 @@ const char kPreloadURL[] = "preloadURL";
 // Enable the node integration.
 const char kNodeIntegration[] = "nodeIntegration";
 
+// Disable the remote module
+const char kDisableRemote[] = "disableRemote";
```

Comment:
Can it be `enableRemoteModule`?
With inverted logic and without confusing `remote`, which can mean anything )

Reply:
I will however have to leave it disableremote in the `<webview>` attributes as they are flags without value for boolean values. The same pattern is used for `webSecurity: !params.disablewebsecurity,`

Reply:
@miniak The `disableremote` attribute is fine, it is (kinda stupid but still) an HTML convention.